index.html

```diff
@@ -4238,29 +4198,31 @@ <h2>DID URL Dereferencing</h2>
     </p>
 
       <section>
-        <h3>DID URL Dereferencing Input Metadata Properties</h3>
+        <h3>DID URL Dereferencing Options</h3>
 
         <p>
 The possible properties within this structure and their possible values SHOULD
 be registered in the DID Specification Registries [[?DID-SPEC-REGISTRIES]].
-This specification defines the following common properties.
+This specification defines the following common properties. This
+specification defines the following common properties for a
+<dfn>DereferencingOptions</dfn> <a data-cite="INFRA#maps">map</a>:
```

Comment:
```suggestion
specification defines the following common properties for the
<dfn>options</dfn> <a href="metadata-structure">metadata structure</a>:
```

The naming in this section should be independent of the TypeScript example.

Reply:
Removed all types and examples and approached testability through a "proof by construction" rather than by providing concrete examples.